frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -197,7 +197,7 @@ class Multiselect extends React.PureComponent<Props, State> {
 
     return (
       <div className="row-widget stMultiSelect" style={style}>
-        <StyledWidgetLabel>
+        <WidgetLabel visible={!!element.label}>
```

Comment:
I remember we had issues with vertical alignment for elements that had a label and those that don't. This should be addressed.

![425491A8-4C0D-4756-81B9-0BE7ABDAE78E](https://user-images.githubusercontent.com/69432/128056516-bd8c0d23-152d-4f66-9cbf-03ab807d2ba0.png)

Current release:

![B01158E3-2684-47C4-B508-F81460E2FAE5](https://user-images.githubusercontent.com/69432/128056611-d6355d27-1abb-4b6f-97e1-b8e5237182f8.png)